src/pages/write-smart-contracts/principals.md

```diff
@@ -61,15 +61,15 @@ example below.
 
 Smart contracts themselves are principals and are represented by the
 smart contract's identifier -- which is the publishing address of the
-contract _and_ the contract's name, e.g.:
+contract _and_ the contract's name, for example:
 
 ```clarity
 'SZ2J6ZY48GV1EZ5V2V5RB9MP66SW86PYKKQ9H6DPR.contract-name
 ```
 
 For convenience, smart contracts may write a contract's identifier in the
 form `.contract-name`. This will be expanded by the Clarity interpreter into
-a fully-qualified contract identifier that corresponds to the same
+a fully qualified contract identifier that corresponds to the same
```

Comment:
Should be fully-qualified

Reply:
this is against the rules and I think for good reasons. [FQDNs](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) don't use the hyphen.

Ultimately, if we believe exceptions like this should be ok, we should start working on our own styleguide. The complexity would blow up quite quickly. I'd rather just use a default for now and make our life easier